### ISSUE TITLE
Console UI: Audit accessibility and make improvements

### ISSUE THREAD

**Issue by alexandec** [department-of-veterans-affairs/va.gov-team#39063]
User Story: in order to make the app accessible to all users an a11y audit should be performed and changes made as needed.

**Comment by alexandec** [comment]
Now that the plugin has a few features implemented, let's check for a11y issues and fix whatever problems we find.

**Comment by laineymajor** [comment]
@alexandec is this something I can close as it's part of a completed epic?

**Comment by alexandec** [comment]
@laineymajor we could either close this ticket and create a new ticket for an accessibility audit, or keep this ticket and move it to a new epic. We should still perform the audit one way or another

**Comment by rmessina1010** [comment]
Console UI work on hold

**Comment by jhonnyoddball** [comment]
Console UI work, closing for now.
